ci/build_python.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+source rapids-env-update
+
+export CMAKE_GENERATOR=Ninja
+
+rapids-print-env
+
+rapids-logger "Begin py build"
+
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+
+# TODO: Remove `--no-test` flag once importing on a CPU
+# node works correctly
+rapids-mamba-retry mambabuild \
+  --no-test \
+  --channel "${CPP_CHANNEL}" \
+  conda/recipes/pylibraft
+
+# TODO: Remove `--no-test` flag once importing on a CPU
+# node works correctly
+rapids-mamba-retry mambabuild \
+  --no-test \
+  --channel "${CPP_CHANNEL}" \
```

Comment:
```suggestion
  --channel "${CPP_CHANNEL}" \
  --channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
```

Since `raft-dask` depends on `pylibraft`, we need to add this extra channel to ensure that the `raft-dask` build uses the artifacts from the previous `pylibraft` build.

This is similar to what we did in `cudf` here: https://github.com/rapidsai/cudf/blob/72361a4a72670f460c074d00c978627159cdc60d/ci/build_python.sh#L18-L45

For reference, `RAPIDS_CONDA_BLD_OUTPUT_DIR` is defined/used in our CI images here: https://github.com/rapidsai/ci-imgs/blob/d8ada6950f26440d4d63fdaabc8bf8af9efb325d/Dockerfile#L104-L107

Reply:
Actually I'll commit this change, but leave this unresolved for visibility. @bdice feel free to mark it as resolved once you view it.